### PR TITLE
fix mistaken info print

### PR DESCRIPTION
**Pull request by zhangxiaoyu-zidif**
**What this PR does / why we need it**:
fix mistaken info print

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by zhangxiaoyu-zidif** [comment]
@humblec 
for approve

File: pkg/volume/glusterfs/glusterfs.go [hunk 1]
```diff
@@ -669,7 +669,7 @@ func (d *glusterfsVolumeDeleter) Delete() error {
 	if err != nil {
 		glog.Errorf("error when deleting endpoint/service :%v", err)
 	} else {
-		glog.V(1).Infof("endpoint: %v and service: %v deleted successfully ", dynamicNamespace, dynamicEndpoint)
+		glog.V(1).Infof("endpoint: [%v/%v] is deleted successfully ", dynamicNamespace, dynamicEndpoint)
```
Comment (humblec) [inline pkg/volume/glusterfs/glusterfs.go]: It would be good to make it as "endpoint/service: [%v/%v] ..." format

**Comment by humblec** [comment]
Thanks @zhangxiaoyu-zidif

**Pull request merged**